integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -530,6 +580,10 @@ fn test_resharding_v3_base(params: TestReshardingParameters) {
         base_epoch_config_store.get_config(base_protocol_version).as_ref().clone();
     base_epoch_config.shuffle_shard_assignment_for_chunk_producers =
         params.shuffle_shard_assignment_for_chunk_producers;
+    // TODO(resharding) Test chunk validators too (would need to change the lines below).
+    base_epoch_config.num_block_producer_seats = params.num_validators;
+    base_epoch_config.num_chunk_producer_seats = params.num_validators;
+    base_epoch_config.num_chunk_validator_seats = params.num_validators;
```

Comment:
Without this shard shuffling test failed because default number of seats was 100, and it treated rpc and archival nodes as validators.

Reply:
indeed.. I've run into the same issue while adding other node types.